test/functional/test_framework/test_node.py

```diff
@@ -269,7 +269,7 @@ def assert_start_raises_init_error(self, extra_args=None, expected_msg=None, mat
                 # Check stderr for expected message
                 if expected_msg is not None:
                     log_stderr.seek(0)
-                    stderr = log_stderr.read().decode('utf-8').strip()
+                    stderr = log_stderr.read().decode('utf-8', 'ignore').strip()
```

Comment:
Hmm, any rationale for this? I'd prefer to throw if the log was not unicode.

Reply:
Done in #14192